### PR TITLE
Add packaging instructions; closes #8

### LINKED ISSUE #8: Packaging
https://forge.glpi-project.org/projects/glpi/wiki/GlpiPackaging

### PR DESCRIPTION
**Pull request by trasher**

### CODE REVIEW

**Comment by trasher** [comment]
See on: http://glpi-developer-documentation.rtfd.io/en/feature-packaging/packaging.html

File: source/packaging.rst [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+   <Directory /usr/share/glpi>
+       Options None
+       AllowOverride None
+
+       # to overwrite default configuration which could be less than recommanded value
+       php_value memory_limit 64M
+
+       Order Deny,Allow
+       Allow from all
```
Comment (remicollet) [inline source/packaging.rst]: this is apache 2.2 only

        Require all granted

File: source/packaging.rst [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+   <?php
+   //[...]
+
+   define('GLPI_FONT_FREESANS',  '/path/to/FreeSans.ttf');
+
+Notes
+-----
+
+This informations are taken from the Fedora/EPEL spec file.
```
Comment (remicollet) [inline source/packaging.rst]: See latest version of the files
* [config_path.path](http://pkgs.fedoraproject.org/cgit/rpms/glpi.git/plain/glpi-9.1-config_path.php)
* [nginx/glpi.conf](http://pkgs.fedoraproject.org/cgit/rpms/glpi.git/plain/glpi-nginx.conf)
* [httpd/glpi.conf](http://pkgs.fedoraproject.org/cgit/rpms/glpi.git/plain/glpi-httpd.conf)

**Comment by trasher** [comment]
Thanks for the review and comments ;)

**Pull request merged**